### DISCUSSION TITLE
Unable to Use Animation Nodes Plugin with Blender 3.6.5

### DISCUSSION THREAD

**Discussion by burhanaksendir** [JacquesLucke/animation_nodes#1933] [Q&A] [answered]
I am experiencing issues using the Animation Nodes plugin with Blender version 3.6.5. The plugin doesn't seem to work as expected.

**Environment:**
- Blender version: 3.6.5
- Operating System: [macOS 14.1.1 ]

Any help or guidance on resolving this issue would be greatly appreciated. Thank you!

**Comment by OmarEmaraDev** [comment]
What is the issue exactly? Do you get an error when enabling the add-on? What is the error?

**Reply by burhanaksendir** [reply]
Hi @OmarEmaraDev,
Thank you. I'm using a Mac with an M1 processor. I have Blender 3.6.5 installed, and I tried to install the Animation Nodes plugin by following these steps:

I downloaded it from here: https://github.com/JacquesLucke/animation_nodes/releases/download/v2.2-for-blender-v2.93-cd-build/animation_nodes_v2_2_macOS_arm_py39.zip
When I try to enable the add-on, I get the following error:

```
There is a Python version mismatch.
You have three options:
1. Try to make Blender use another Python version.
   (Blender 2.93 officially uses Python 3.9.x)
2. Compile Animation Nodes yourself using the correct Python version.
   (Look in the developer manual for more information)
3. Create an issue on Github and ask if someone can create a build for you.
```

I would appreciate it if you could help me with this issue.

**Reply by OmarEmaraDev** [reply, marked as answer]
Unfortunately, we don't have new builds for M1 Mac. See https://github.com/JacquesLucke/animation_nodes/issues/1926.
Can you try this build by the community?
https://github.com/JacquesLucke/animation_nodes/issues/1887#issuecomment-1356246203

**Reply by burhanaksendir** [reply]
Thank you. I solved the problem by downloading the file you shared. Now it's working perfectly fine.

https://github.com/JacquesLucke/animation_nodes/files/10251739/animation_nodes_v2_3_macOS_py310.zip

**Reply by 8bitsboy** [reply]
Man @OmarEmaraDev you have no idea how much time you're saving me.
I'm animation a keyboard with line art on it. This was so fucking slow on 2.9.3 lol
Now I can use animation node with Blender 4. THANK YOU!
